website/content/en/docs/deploy-guide.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Development Guide"
+linkTitle: "Development Guide"
+weight: 50
+---
+
+# Resources Karpenter Expects/Creates
+
+- Cluster on AWS
+    - min version 1.16? 1.18?
+    - must be EKS?
+- IAM Roles
+    - `KarpenterNode` -- run containers, configure networking
+    - `Karpenter` -- provision new instances
+    - IRSA vs kube2iam?
+- Service Accounts
```

Comment:
I might separate AWS IAM auth concepts from RBAC concepts. The karpenter service account is an RBAC concept.